tests/e2e/app-specs/verify-extensions.e2e-spec.ts

```diff
@@ -0,0 +1,49 @@
+import { initAuth } from "../helpers/authentication-manager";
+import { ItemList } from '../app-elements/item-list.component';
```

Comment:
We are supposed to use only double quotes per our guidelines.
@dechoD @veliniv maybe we are missing tslint